IntegrationTests/Runner/Scenarios/RUM/StopSessionScenario/KioskSendEventsViewController.swift

```diff
@@ -36,7 +36,7 @@ internal class KioskSendEventsViewController: UIViewController {
 
         let simulatedResourceKey = "/resource/1"
         let simulatedResourceRequest = URLRequest(url: URL(string: "https://foo.com/resource/1")!)
-        let simulatedResourceLoadingTime: TimeInterval = 0.1
+        let simulatedResourceLoadingTime: TimeInterval = 0.5
```

Comment:
can this be a real issue?

for example if a customer uses start and stop resource APIs right after.
```swift
        rumMonitor.startResource(
            resourceKey: simulatedResourceKey,
            request: simulatedResourceRequest
        )
        rumMonitor.stopResource(
                resourceKey: simulatedResourceKey,
                response: HTTPURLResponse(
                    url: simulatedResourceRequest.url!,
                    statusCode: 200,
                    httpVersion: nil,
                    headerFields: ["Content-Type": "image/png"]
                )!
            )
```

Reply:
This won't be an issue for users, if they start and stop the resource, it will just not keep the view as active and it's completely fine.
The flakiness comes from what we expect in the integration test, where we assert that the view is first active then inactive when resources are done and the session is stoped.

Reply:
perfect, thanks for quick turnaround.